docs/source/expansion.md

```diff
@@ -36,6 +36,9 @@ Authorization: Basic YWRtaW46c2VjcmV0
     {
       "@id": "http://localhost:55001/plone/front-page/@navigation"
     },
+    {
+      "@id": "http://localhost:55001/plone/front-page/@navroot"
+    },
```

Comment:
Does `@site` also need to be added here?

Reply:
I am not sure about adding `@site` as an expandable endpoint..., what do you think @cekk?